### DISCUSSION TITLE
Is it possible to update an existing verifiable credential?

### DISCUSSION THREAD

**Discussion by adnan-alam-rfera** [decentralized-identity/veramo#830] [Q&A]
Hi,
I would like to know that is it possible to update the information of existing verifiable credentials?
For example, I've created a verifiable credential for user profile with name, username, email, etc. 
Now, the user email has been changed so in this case how to update the email in the previously created verifiable credential?

**Comment by mirceanis** [comment]
Its not possible to modify a credential because the data in it is signed, so any modification would invalidate the signature.

If you are the issuer of the original credential, then you can issue a new credential with the new data and send it to the user.

In some cases, the old credential should be revoked, but it depends on the use-case.
We are working on a revocation API for Veramo and a corresponding `credentialStatus` check. If you have ideas or requirements for this, feel free to voice them.
